core/src/main/java/com/godaddy/commerce/core/config/LoggerStartupListener.java

```diff
@@ -0,0 +1,57 @@
+package com.godaddy.commerce.core.config;
+
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.Logger;
+import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.classic.spi.LoggerContextListener;
+import ch.qos.logback.core.Context;
+import ch.qos.logback.core.spi.ContextAwareBase;
+import ch.qos.logback.core.spi.LifeCycle;
+
+import java.util.Optional;
+
+public class LoggerStartupListener extends ContextAwareBase implements LoggerContextListener,
+    LifeCycle {
+
+  private boolean started = false;
+
+  @Override
+  public void start() {
+    if (started) return;
+    String logLevel = Optional.ofNullable(System.getenv("LOG_LEVEL")).orElseGet(() -> "INFO");
```

Comment:
> [<img alt="jburke" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.secureserver.net/jburke) **Authored by [jburke](https://github.secureserver.net/jburke)**
_<time datetime="2020-12-17T01:01:32Z" title="Thursday, December 17th 2020, 2:01:32 am +01:00">Dec 17, 2020</time>_

---


```suggestion
    String logLevel = Optional.ofNullable(System.getenv("COMMERCE_LOG_LEVEL")).orElseGet(() -> "INFO");
```